build.sbt

```diff
@@ -22,13 +21,13 @@ lazy val testDependencies = Seq(
 
 lazy val ItTest = config("it").extend(Test)
 
-lazy val scala213 = "2.13.11"
-lazy val scala212 = "2.12.18"
+lazy val scala3 = "3.3.1"
+lazy val scala213 = "2.13.12"
```

Comment:
Maybe good to call out in the PR that the project won't be cross compiled for Scala v2.12.